File: One/Daytime.php
<?php

include_once 'Overtime.php';

class Daytime extends Overtime {
    private const PERCENTAGE = 1.25;

    public function calculate(float $baseSalary, int $hoursWorked): float {
        $normalHourValue = $baseSalary / (30 * 8); // Assuming 30 days and 8 hours per day
        return $normalHourValue * self::PERCENTAGE * $hoursWorked;
    }
}

?>

File: java/One/One/Daytime.java
public class Daytime extends Overtime {
    private static final double PERCENTAGE = 1.25;

    @Override
    public double calculate(double baseSalary, int hoursWorked) {
        double normalHourValue = baseSalary / (30 * 8);
        return normalHourValue * PERCENTAGE * hoursWorked;
    }
}   